pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.5</version>
+        <version>3.56</version>
```

Comment:
Why not 3.57¿

Reply:
IDE autocompletion 👅 
```suggestion
        <version>3.57</version>
```

Reply:
Sorry was on github mobile where suggestion is not possible yet, or at least very tedious to type the suggestion 😅